react/components/EXPERIMENTAL_Table/Sections/Tbody.tsx

```diff
@@ -71,4 +71,4 @@ const FowardedTbody: ComposableTbody = forwardRef(Tbody)
 
 FowardedTbody.Row = Row
 
-export default FowardedTbody
+export default React.memo(FowardedTbody)
```

Comment:
This is needed because even adding a `useMemo` in all contexts it will re-render if `Table` gets called